conans/test/functional/generators/pkg_config_test.py

```diff
@@ -138,7 +136,7 @@ def package_info(self):
         pc_path = os.path.join(client.current_folder, "MyLib.pc")
         self.assertTrue(os.path.exists(pc_path))
         pc_content = load(pc_path)
-        self.assertIn("-Wl,-rpath=\"${libdir}\"", pc_content)
+        self.assertIn("-Wl,-rpath,\"${libdir}\"", pc_content)
```

Comment:
These changes, was it a bug? Maybe this is worth an entry in the changelog.

Reply:
There was another PR for this issue, not this one, I am surprised this is here, need to check.

Reply:
https://github.com/conan-io/conan/issues/7715

Reply:
The PR https://github.com/conan-io/conan/pull/7716/files contains this change, and this was merged to develop, and this PR merged develop, so I think it might be a glitch of Github diff